tests/UserLogonStateEventsTests.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * User Logon State Change Test Definitions
+ *
+ * Copyright 2013-2015 OCAD University
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * The research leading to these results has received funding from the European Union's
+ * Seventh Framework Programme (FP7/2007-2013)
+ * under grant agreement no. 289016.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+"use strict";
+
+var fluid = require("infusion"),
+    jqUnit = fluid.require("node-jqunit", require, "jqUnit"),
+    gpii = fluid.registerNamespace("gpii");
+
+fluid.require("%universal");
+
+require("./shared/UserLogonStateChangeTestDefs.js");
+fluid.registerNamespace("gpii.tests.userLogonEvents");
+
+gpii.tests.userLogonEvents.expectedEvent = function () {
+    jqUnit.assertTrue("Expected event fired", true);
+};
+
+gpii.tests.userLogonEvents.testDefs = [{
+    name: "Testing events related to login and logout",
```

Comment:
Add words here to indicate that it is specifically the proximityTriggered endpoint that is being tested